aspnetcore/host-and-deploy/windows-service.md

```diff
@@ -110,18 +157,22 @@ The following minimum changes are required to set up an existing ASP.NET Core pr
 
    * `{SERVICE NAME}` &ndash; The name to assign to the service in [Service Control Manager](/windows/desktop/services/service-control-manager).
    * `{PATH}` &ndash; The path to the service executable.
-   * `{DOMAIN}` (or if the machine isn't domain joined, the local machine name) and `{USER ACCOUNT}` &ndash; The domain (or local machine name) and user account under which the service runs. Do **not** omit the `obj` parameter. The default value for `obj` is the [LocalSystem account](/windows/desktop/services/localsystem-account) account. Running a service under the `LocalSystem` account presents a significant security risk. Always run a service under a user account with restricted privileges on the server.
+   * `{DOMAIN}` &ndash; The domain of a domain-joined machine. If the machine isn't domain-joined, the local machine name.
+   * `{USER ACCOUNT}` &ndash; The user account under which the service runs.
    * `{PASSWORD}` &ndash; The user account password.
 
-   In the following example:
+   > [!WARNING]
+   > Do **not** omit the `obj` parameter. The default value for `obj` is the [LocalSystem account](/windows/desktop/services/localsystem-account) account. Running a service under the `LocalSystem` account presents a significant security risk. Always run a service with a user account that has restricted privileges.
+
+   In the following example for the sample app:
 
    * The service is named **MyService**.
-   * The published service resides in the *c:\\svc* folder. The app executable is named *AspNetCoreService.exe*. The `binPath` value is enclosed in straight quotation marks (").
+   * The published service resides in the *c:\\svc* folder. The app executable is named *SampleApp.exe*. Enclose the `binPath` value in straight quotation marks (").
```

Comment:
What's a straight quotation mark? Can't tell if this asking me to use double quotes or single quotes.

Reply:
Straight `"` != single `'`. I don't think smart/curly quotes (`“ ”`) work. I'll remove "straight." If someone tries to use them and it :boom: :cry:, I'm sure they'll ask.